packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  getFocusStyle,
+  HighContrastSelector
+} from '../../Styling';
+import {
+  ILinkStyleProps,
+  ILinkStyles
+} from './Link.types';
+
+export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
+  const { disabled, theme } = props;
+  const { palette, semanticColors } = theme;
+
+  return {
+    root: [
+      'ms-Link',
+      {
+        color: semanticColors.linkText,
+        margin: 0,
```

Comment:
can we move all these reset styles to the "button&" block since they're not needed for the <a>?

Reply:
I matched the previous SCSS here, but might as well fix it while we're making changes. I'll get those moved.